homeassistant/components/bryant_evolution/__init__.py

```diff
@@ -0,0 +1,38 @@
+"""The Bryant Evolution integration."""
+
+from __future__ import annotations
+
+import logging
+
+from evolutionhttp import BryantEvolutionClient
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, Platform
+from homeassistant.core import HomeAssistant
+
+from .const import CONF_SYSTEM_ID, CONF_ZONE_ID
+
+PLATFORMS: list[Platform] = [Platform.CLIMATE]
+
+type BryantEvolutionConfigEntry = ConfigEntry[BryantEvolutionClient]  # noqa: F821
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: BryantEvolutionConfigEntry
+) -> bool:
+    """Set up Bryant Evolution from a config entry."""
+    if not hasattr(entry, "runtime_data") or entry.runtime_data is None:
+        entry.runtime_data = BryantEvolutionClient(
+            entry.data[CONF_HOST], entry.data[CONF_SYSTEM_ID], entry.data[CONF_ZONE_ID]
+        )
```

Comment:
Can we check if the device responds before moving on to setting up entities?

Reply:
We do this in the config validation; do you also want it here>

Reply:
Yes, we only want to setup the integration when we know the device responds. You should then raise ConfigEntryNotReady and it will show in a user-friendly way that the integration doesn't load yet

Reply:
Added to climate.py:async_setup_entry